setup.cfg

```diff
@@ -17,7 +17,7 @@ exclude_lines =
 [flake8]
 # Flake8, PyFlakes, etc
 max-line-length = 119
-ignore = E203,W605,W503
+ignore = E203,W605,W503,B007,B009,B011,B014,B017
```

Comment:
These are other errors brought in by `flake8-bugbear` - most of which are quick improvements that I will introduce in subsequent changes, but did not want to bring into this PR.